Essentials/src/com/earth2me/essentials/EssentialsEntityListener.java

```diff
@@ -152,6 +153,10 @@ public void onPlayerDeathEvent(final PlayerDeathEvent event) {
         if (!ess.getSettings().areDeathMessagesEnabled()) {
             event.setDeathMessage("");
         }
+        if (ess.getSettings().infoAfterDeath()) {
+            final Location loc = user.getLocation();
+            user.sendMessage(tl("infoAfterDeath", loc.getWorld().getName(), loc.getBlockX(), loc.getBlockY(), loc.getBlockZ()));
```

Comment:
This is a tiny nitpick but IMO we should send this message *before* the `/back` message, rather than after it.